script/publishing-api-sync-checks/detailed_guide_sync_check.rb

```diff
@@ -0,0 +1,78 @@
+detailed_guides = DetailedGuide.latest_published_edition
+check = DataHygiene::PublishingApiSyncCheck.new(detailed_guides)
+
+check.add_expectation("format") do |content_store_payload, _|
+  content_store_payload["format"] == "detailed_guide"
+end
+
+check.add_expectation("base_path") do |content_store_payload, record|
+  content_store_payload["base_path"] == record.search_link
+end
+
+check.add_expectation("title") do |content_store_payload, record|
+  content_store_payload["title"].strip == record.title.strip
```

Comment:
I think we should only strip the Whitehall model's title, this means we test that the title is being stripped correctly before being sent to publishing api.